exporters/sender/okhttp/src/main/java/io/opentelemetry/exporter/sender/okhttp/internal/RetryInterceptor.java

```diff
@@ -75,9 +76,10 @@ public Response intercept(Chain chain) throws IOException {
       if (attempt > 0) {
         // Compute and sleep for backoff
         // https://github.com/grpc/proposal/blob/master/A6-client-retries.md#exponential-backoff
-        long upperBoundNanos = Math.min(nextBackoffNanos, retryPolicy.getMaxBackoff().toNanos());
-        long backoffNanos = randomLong.get(upperBoundNanos);
-        nextBackoffNanos = (long) (nextBackoffNanos * retryPolicy.getBackoffMultiplier());
+        long currentBackoffNanos =
+            Math.min(nextBackoffNanos, retryPolicy.getMaxBackoff().toNanos());
+        long backoffNanos = (long) (randomJitter.get() * currentBackoffNanos);
+        nextBackoffNanos = (long) (currentBackoffNanos * retryPolicy.getBackoffMultiplier());
```

Comment:
Should also update the implementation in [JdkHttpSender](https://github.com/open-telemetry/opentelemetry-java/blob/9c1b9f609fe6ece64923c126990f57981b7171fc/exporters/sender/jdk/src/main/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSender.java#L216-L229). 

I know its not ideal that there are two implementations.. Maybe worth adding a utility function to `RetryPolicy` that computes the backoff for a given attempt `N`. Signature might look like:

```
public long computeBackoffNanosForAttempt(int attempt, Random randomSource) {...}
```

It wouldn't be as efficient as the current implementation, but...

- its such a tiny amount of compute that who cares
- the compute is trivial compared to the overall cost of preparing and executing and HTTP request

Reply:
Thanks, Added code for JdkHttpSender

I did not consider adding that method to calculate a backoff delay time
looking at the code I would say we can build more abstractions, but not sure it's really helpful so let's probably move on with duplicated approach as we had before